site/js/util/api.js

```diff
@@ -0,0 +1,39 @@
+// wrapper around window.fetch
+
+async function fetchHelper(host, path, fetchConfig = {}) {
```

Comment:
This should throw an error if `host` is not given. (Actually, it should also throw an error if path isn't given, but a missing host argument is more likely - the path is usually a static string given straight into the function call, while the host is usually gotten from some variable, which might have an undefined/null value.)